components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -370,13 +498,19 @@ class SitePermissionsFeature(
         return false
     }
 
+    private val PermissionRequest.host get() = uri?.toUri()?.host ?: ""
+    private val Permission.isBackCamera get() = desc?.contains("back") ?: false
+    private val Permission.isFrontCamera get() = desc?.contains("front") ?: false
```

Comment:
This code existed before this PR, but I wonder: Is the description localized? Are there other descriptions? Can't we just use the descriptions here instead of translating them to other strings?

Reply:
> This code existed before this PR

I saw what the GV team did in [geckoview_example](https://dxr.mozilla.org/mozilla-central/rev/c2593a3058afdfeaac5c990e18794ee8257afe99/mobile/android/geckoview_example/src/main/java/org/mozilla/geckoview_example/GeckoViewActivity.java#615).

```java
        private String[] normalizeMediaName(final MediaSource[] sources) {
            if (sources == null) {
                return null;
            }

            String[] res = new String[sources.length];
            for (int i = 0; i < sources.length; i++) {
                final int mediaSource = sources[i].source;
                final String name = sources[i].name;
                if (MediaSource.SOURCE_CAMERA == mediaSource) {
                    if (name.toLowerCase(Locale.ENGLISH).contains("front")) {
                        res[i] = getString(R.string.media_front_camera);
                    } else {
                        res[i] = getString(R.string.media_back_camera);
                    }
                } else if (!name.isEmpty()) {
                    res[i] = name;
                } else if (MediaSource.SOURCE_MICROPHONE == mediaSource) {
                    res[i] = getString(R.string.media_microphone);
                } else {
                    res[i] = getString(R.string.media_other);
                }
            }

            return res;
        }
```


> Is the description localized?

I have to investigate because right now this is the only thing that help us to identify between one or another. And if it is localized can break that logic.  

>  Are there other descriptions?

So far, I just saw these two, and one for the microphone. I have to investigate.
In the Fenix, mock-up I saw additional microphone devices but in code I just get one.
![image](https://user-images.githubusercontent.com/773158/53966671-9a1d7b80-40c1-11e9-843c-3f832120ee50.png)



> Can't we just use the descriptions here instead of translating them to other strings?

I think it won't be ideal for  apps, for instance Fenix has its own strings
![image](https://user-images.githubusercontent.com/773158/53966843-07c9a780-40c2-11e9-9a9d-211ecd465d14.png)

![image](https://user-images.githubusercontent.com/773158/53966671-9a1d7b80-40c1-11e9-843c-3f832120ee50.png)